core/templates/services/playthrough-issues-backend-api.service.ts

```diff
@@ -19,98 +19,81 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { HttpClient, HttpResponse } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { throwError } from 'rxjs';
 
-import { PlaythroughIssueObjectFactory, PlaythroughIssue } from
+import { PlaythroughIssue, PlaythroughIssueObjectFactory } from
   'domain/statistics/PlaythroughIssueObjectFactory.ts';
 import { ServicesConstants } from 'services/services.constants.ts';
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service.ts';
 
-@Injectable({
-  providedIn: 'root'
-})
+@Injectable({ providedIn: 'root' })
```

Comment:
Could we please revert this to original since we've been using the original pattern everywhere?

Reply:
Done.